code/modules/species/station/human/human_mercurial.dm

```diff
@@ -0,0 +1,21 @@
+/datum/species/human/mercurial
+	name = SPECIES_MERCURIAL
+	name_plural = "Mercurials"
+	blurb = "Heavier and stronger than a baseline human, the Mercurial gene-line stems from \
+	modifications made to adapt colonists to the incandecent skies of Mercury.<br><br>Mercurials have \
```

Comment:
Mercury doesn't have a sky, so unless this references the sun you may have mixed it up with Venus.